bin/supply

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+# bin/supply <build-dir> <cache-dir> <deps-dir> <index>
+
+BUILD_DIR=$1
```

Comment:
It would be helpful to add a quick comment for each of the directories and their purpose/role, either here or in the README.

Reply:
Done!